machine1/dbus_test.go

```diff
@@ -58,6 +60,17 @@ func mustCreateTestProcess(machineName string) (pid int) {
 	return int(mainPID)
 }
 
+func mustCreateTestImage(imagePath string) error {
```

Comment:
I think this is small enough that it can be simply inlined into `TestImages` (which is the only consumer). Also the `defer os.Remove` should in-between `Create` and `Truncate` to avoid potentially leaking empty files.